app/models/article.rb

```diff
@@ -1,4 +1,10 @@
 class Article < ApplicationRecord
+  # From the recommendation of the strong migrations gem:
+  #
+  # Active Record caches attributes, which causes problems when
+  # removing columns. Be sure to ignore the column:
+  self.ignored_columns = ["privileged_users_reaction_points_sum"]
```

Comment:
Usually, this is only used when removing a column, not when adding a new one. Ignoring it like this means it will not be available on the Ruby object, is this what you want?

```ruby
Article.ignored_columns
#=> ["privileged_users_reaction_points_sum"]
Article.first.privileged_users_reaction_points_sum
# NoMethodError: undefined method `privileged_users_reaction_points_sum' for....
```

The problem this tries to solve is the following: you deploy a migration removing a column, but it's still in the AR attributes cache. To avoid this you first deploy a change that ignores the column, which will remove it from the cache. Then you deploy the removal migration. In the same PR you can also remove the `self.ignored_columns` part again because the column is already out of the cache.

Sorry if I misunderstood something, it's just extremely unusual to see a PR that adds a new column and ignores it immediately. Did you maybe have some problems with a local down migration? The way to fix this is not by adding this line but by temporarily changing the value of `StrongMigrations.check_down` to false in your local environment.

Reply:
Thanks for the additional information.  I added it without fully understanding @djuber's above message.  I'm removing it from this commit.

Reply:
Indeed - I also didn't understand the warning from strong migrations (and I was performing a down migration when I got the error).

Thanks!